src/pages/HomePage.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import Home from '../features/Home/Home'
+
+function HomePage() {
```

Comment:
Not sure if we're missing any rule or this is because of our typescript packages versioning, but as far as I remember, React `.tsx` components need a return type (usually `JSX.Element`)

Reply:
changed to arrow function